packages/api-clients/open-api/agreementApi.yml

```
@@ -1270,6 +1343,13 @@ components:
       required:
         - results
         - totalCount
+    hasCertifiedAttributes:
```

Also here, let's capitalize, as did in BFF
```suggestion
    HasCertifiedAttributes:
```